packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -3,7 +3,8 @@
         :is-open="isOpen"
         data-test-id="checkout-issue-modal"
         has-overlay
-        @close="handleClose">
+        v-bind="$attrs"
+        v-on="$listeners">
```

Comment:
Nice. I guess this is where me being new to Vue got me. 🙈

Reply:
So it was actually the `v-if` that was breaking it mostly. The mega modal wasn't picking up the `hasOpened` value as that `if` [here](https://github.com/justeat/fozzie-components/pull/845/files#diff-3f7bda367e6ae2f5c141559c1aec5722d6606ee4d895f7d44e9837c20f7efb3cL4) stopped it being populated so the close event would never execute properly :)